### PR TITLE
[FAQ 4] Add content for help resources

### PR DESCRIPTION
**Pull request by okkays**
- 🌎 This content goes in the new help/FAQ page
- ✅ This commit adds a list of links between topics and relevant standards sections

Screenshot 2024-11-22 at 3 16 27 PM

Screenshot 2024-11-22 at 3 16 41 PM

### CODE REVIEW

**Comment by okkays** [comment]
(Pending feedback from https://github.com/usdoj-crt/beta.ADA.gov-Project-Management/issues/1036#issuecomment-2494651934)

File: _config.yml [hunk 1]
```diff
@@ -71,6 +71,17 @@ primary_navigation:
...
+    - name:
+        en: Guidance & Resource Materials
+        en-short: Resources
+        es: Guidance & Resource Materials
+      url: "/resources/"
+    - name:
+        en: Help
+        en-short: Help
+        es: Help
+      url: "/help/"
```
Comment (Rwolfe-Nava) [inline _config.yml:84]: Just checking: are `en` and `es` here supposed to be the same? Is there some other system that is handling the translation for us?

**Review by Rwolfe-Nava — APPROVED** [review]
Just a small question regarding translation, but otherwise this looks good to me

File: _config.yml (hunk 1 above)
Comment (okkays) [reply _config.yml:84]: This is where we'd put the translations, but that would be separate work to procure those - even for a single word like this - so we'll default to en for now.

**Pull request merged**